### ISSUE TITLE
about the problem of overweighting of lower hierarchies in 3.3 of ImageNet-21K Pretraining for the Masses

### ISSUE THREAD

**Issue by eeyzs1** [Alibaba-MIIL/ImageNet21K#73]
Hi Tal，
I was noticed below part conflicting with the functions(2,3) provided, while reading  3.3 Semantic Softmax Training Scheme of ImageNet-21K Pretraining for the Masses:

![Image](https://github.com/user-attachments/assets/657c9683-746c-4519-a7c0-ef8953b8bd53)

WIth the functions(2,3) provided, the lower hierarchies will have a smaller Ok((O1 = N0)<(O2=N0+N1)), thus cause Wk((W1=1/O1)>(W2=1/O2)) to be larger, so that conflicted with the words I was labeled in the picture.
I was also read the source code, it seems have the same conflict if I didnt misunderstand the code or ignored sth.

Regards,
Zongyuan Sui

**Comment by eeyzs1** [comment]
I was tried to send email, but it was rejected, so I came here and raise a ticket.

**Comment by eeyzs1** [comment]
I was tried to send email, but it was rejected, so I came here and raise a ticket.
